Calendar of the Archdiocese of Lyon in France

emagnier opened the pull request:
Here is a new calendar for the archdiocese of Lyon in France. 

I know we need to think about how organizing and defining all these calendars (continents, countries, dioceses, cities...), but in the meantime, here is a new one, ready to be reviewed 😄 

## **Sources** 

Proper Missal of the Archdiocese of Lyon (2004).

## **Added calendar items** (and corresponding martyrology metadata):
- `gregory_x_pope`: https://en.wikipedia.org/wiki/Pope_Gregory_X
- `claudine_thevenet_religious`: https://en.wikipedia.org/wiki/Claudine_Th%C3%A9venet
- `jean_pierre_neel_priest`: https://fr.wikipedia.org/wiki/Jean-Pierre_N%C3%A9el
- `baldomerus_of_lyon_religious`: https://fr.wikipedia.org/wiki/Galmier_de_Lyon
- `nicetius_of_lyon_bishop`: https://en.wikipedia.org/wiki/Nicetius_of_Lyon
- `epipodius_and_alexander_of_lyon_martyrs`: https://en.wikipedia.org/wiki/Epipodius_and_Alexander
  - which include `epipodius_of_lyon_martyr` and `alexander_of_lyon_martyr`
- `jean_louis_bonnard_priest`: https://en.wikipedia.org/wiki/Jean-Louis_Bonnard
- `our_lady_of_fourviere`: https://en.wikipedia.org/wiki/Basilica_of_Notre-Dame_de_Fourvi%C3%A8re
- `marcellin_champagnat_priest`: https://en.wikipedia.org/wiki/Marcellin_Champagnat
- `john_francis_regis_priest`: https://en.wikipedia.org/wiki/John_Francis_Regis
- `innocent_v_pope`: https://en.wikipedia.org/wiki/Pope_Innocent_V
- `jacques_jules_bonnaud_martyr_and_companions_martyrs`: https://fr.wikipedia.org/wiki/Jacques_Bonnaud
  - which include `jacques_jules_bonnaud_martyr` and `companions_martyrs`
- `justus_of_lyon_bishop`: https://en.wikipedia.org/wiki/Justus_of_Lyon
- `frederic_ozanam_founder`: https://en.wikipedia.org/wiki/Fr%C3%A9d%C3%A9ric_Ozanam
- `maurice_of_agaunum_and_companions_martyrs`: https://en.wikipedia.org/wiki/Saint_Maurice
  - which include `maurice_of_agaunum_martyr` and `companions_martyrs`
- `therese_couderc_virgin`: https://en.wikipedia.org/wiki/Th%C3%A9r%C3%A8se_Couderc
- `annemund_of_lyon_bishop`: https://en.wikipedia.org/wiki/Annemund
- `antoine_chevrier_priest`: https://en.wikipedia.org/wiki/Antoine_Chevrier
- `the_penitent_thief`: https://en.wikipedia.org/wiki/Penitent_thief
- `viator_of_lyon`: https://en.wikipedia.org/wiki/Viator_of_Lyons
- `dedication_of_the_cathedral_saint_john_of_lyon_france`: https://en.wikipedia.org/wiki/Lyon_Cathedral
- `all_the_holy_bishops_of_the_archdiocese_of_lyon`
- `all_saints_of_the_archdiocese_of_lyon`
- `eucherius_of_lyon_bishop`: https://en.wikipedia.org/wiki/Eucherius_of_Lyon

## **Modified existing calendar items:**
- `francis_de_sales_bishop`
  - Specific localization string for this calendar
  - Added a new title, specific to this archdiocese
- `pothinus_of_lyon_bishop_blandina_of_lyon_virgin_and_companions_martyrs`
  - Specific localization string and precedence for this calendar
  - Titles are redefined for `pothinus_of_lyon_bishop`, specifically to this archdiocese 
 - `marcellinus_of_rome_and_peter_the_exorcist_martyrs`
   - Moved to 6/4
- `irenaeus_of_lyon_bishop`
  - Specific localization string and precedence for this calendar
  - Added companions
  - Titles are redefined for `irenaeus_of_lyon_bishop`, specifically to this archdiocese 
- `john_mary_vianney_priest`
  - Specific precedence for this calendar
- `anthony_mary_claret_bishop`
  - Moved to 10/26

## **Added 2 new Patron type:**
- `PatronOfTheLyonClergy`
- `PatronOfTheCityOfLyon`

Note: adding `Patrons` (as well as `Titles`) here isn't ideal. We should think about a better way to store these kinds of metadata in the future.

## **Added 2 new martyrology metadata properties:**
- `dateOfBirth?: SaintDateDef;`
- `dateOfBirthIsApproximative?: boolean;`

emagnier committed:
data: add the calendar of the archdiocese of lyon

tukusejssirs commented on lib/catalog/martyrology.ts:
```diff
@@ -352,6 +365,13 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
     },
+    antoine_chevrier_priest: {
+      canonizationLevel: CanonizationLevels.Blessed,
+      name: 'Antoine Chevrier',
```
Anglicise the first name of Anthody Chevrier ([src](https://leprado.org/wp-content/uploads/2018/12/Mass-of-F.Chevrier.pdf)).

```suggestion
    anthony_chevrier_priest: {
      canonizationLevel: CanonizationLevels.Blessed,
      name: 'Anthony Chevrier',
```

tukusejssirs commented on lib/catalog/martyrology.ts:
```diff
@@ -443,6 +463,13 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
       name: 'Aurelia',
       titles: [Titles.Virgin],
     },
+    baldomerus_of_lyon_religious: {
+      canonizationLevel: CanonizationLevels.Saint,
+      name: 'Saint Galmier',
+      titles: [Titles.Religious],
```
Was he really a religious? According to [this](https://catholicreadings.org/saint-baldomerus-of-saint-justus-saint-of-the-day-february-27/), he was a blacksmith and a sub-deacon. Is he really celebrated as _religious_?

emagnier replied:
According to the short biography from the proper missal, yes, he was a religious (probably a monk).

tukusejssirs commented on lib/catalog/martyrology.ts:
```diff
@@ -802,6 +830,12 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
+      canonizationLevel: CanonizationLevels.Saint,
+      name: 'Claudine Thévenet',
+      dateOfDeath: 1837,
+      titles: [Titles.Religious],
```
Add religious name to the key name.

According to [Wikipedia](https://en.wikipedia.org/wiki/Claudine_Th%C3%A9venet), she was a foundress; is she not celebrated with this title in the Archdiocese of Lyon?

```suggestion
    marie_of_saint_ignatius_claudine_thevenet_religious: {
      canonizationLevel: CanonizationLevels.Saint,
      name: 'Claudine Thévenet',
      dateOfDeath: 1837,
      titles: [Titles.Religious],
```

emagnier replied:
In the proper missal, only "Religious" is specified for her title. 
About being a foundress is only mentioned in her short biography in the missal.

tukusejssirs commented on lib/catalog/martyrology.ts:
```diff
@@ -1818,6 +1878,13 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
+    jacques_jules_bonnaud_martyr: {
+      canonizationLevel: CanonizationLevels.Blessed,
+      name: 'Jacques Jules Bonnaud',
+      titles: [Titles.Martyr],
```
According to [Wikipedia](https://fr.wikipedia.org/wiki/Jacques_Bonnaud), he was a priest, therefore the key should be renamed to `jacques_jules_bonnaud_priest` and `Titles.Priest` title added?

```suggestion
    jacques_jules_bonnaud_priest {
      canonizationLevel: CanonizationLevels.Blessed,
      name: 'Jacques Jules Bonnaud',
      titles: [Titles.Priest, Titles.Martyr],
```

tukusejssirs commented on lib/catalog/martyrology.ts:
```diff
@@ -3776,6 +3885,10 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
       name: 'Teresa of Portugal',
       titles: [Titles.Religious],
     },
+    the_penitent_thief: {
```
No initial definite article in the key names.

```suggestion
    penitent_thief: {
```

tukusejssirs replied:
@emagnier, did you remove this item from `martyrology.ts`? I can’t find it anymore in the latest commit. 🤔

tukusejssirs commented on lib/catalog/martyrology.ts:
```diff
@@ -3796,6 +3909,13 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
+    therese_couderc_virgin: {
+      canonizationLevel: CanonizationLevels.Saint,
+      name: 'Thérèse Couderc',
+      titles: [Titles.Virgin],
```
Add first name.

Is she really celebrated as _virgin_? According to [Wikipedia](https://en.wikipedia.org/wiki/Th%C3%A9r%C3%A8se_Couderc), she was a co-founder of Sisters of the Cenacle, therefore she could be considered a religious.

```suggestion
    therese_marie_victoire_couderc_virgin: {
      canonizationLevel: CanonizationLevels.Saint,
      name: 'Thérèse Couderc',
      titles: [Titles.Virgin],
```

tukusejssirs commented on lib/constants/martyrology-metadata.ts:
```diff
@@ -106,6 +106,8 @@ export enum PatronTitles {
   PatronessOfThePhilippines = 'PATRONESS_OF_THE_PHILIPPINES',
   PatronessOfTheProvinceOfQuebec = 'PATRONESS_OF_THE_PROVINCE_OF_QUEBEC',
   PatronessOfTheUSA = 'PATRONESS_OF_THE_U_S_A',
+  PatronOfTheLyonClergy = 'PATRON_OF_THE_LYON_CLERGY',
```
It might sound more natural as `PatronOfTheClergyOfLyon`.

Another thing to consider is to define what is Lyon: a city or an archdiocese; therefore consider renaming this to `PatronOfTheClergyOfTheArchdioceseOfLyon` or `PatronOfTheClergyOfTheCityOfLyon`.

emagnier replied:
According to the celebration description in the proper diocesan missal, it indeed refers to the Archdiocese of Lyon (and not the City of Lyon). Thank you for pointing that out.

tukusejssirs commented on lib/locales/en.ts:
```diff
@@ -171,9 +171,11 @@ export const locale: Locale = {
     all_saints_of_the_archdiocese_of_paris: 'All Saints of the Archdiocese of Paris',
     all_saints_of_the_diocese_of_saint_denis: 'All Saints of the Diocese of Saint-Denis',
     all_saints_of_wales: 'All Saints of Wales',
+    all_the_holy_bishops_of_the_archdiocese_of_lyon: 'All the Holy Bishops of the Archdiocese of Lyon',
```
I am not 100 % sure on this one, but consider removing the definite article before `Bishops`; cf:
- Saint [Anatalus](http://www.gcatholic.org/dioceses/diocese/mila0.htm#47488) and All Holy Bishops of Milan ([src](http://www.gcatholic.org/calendar/2024/Ambrosian-en.htm));
- all holy Bishops and Confessors ([src](https://en.wikipedia.org/wiki/Litany_of_the_Saints#Traditionalist_Catholic_practice));
- All Holy Bishops of Jerusalem ([src](https://www.lpj.org/cached_uploads/view/2021/12/17/17-5-all-bishops-of-jer-1639754749.pdf)).

emagnier replied:
Indeed, it looks better without the `the`. I've updated the key and `en`.

tukusejssirs commented on lib/locales/en.ts:
```diff
@@ -211,6 +214,7 @@ export const locale: Locale = {
     anthony_of_saint_anne_galvao_priest: 'Saint Anthony of Saint Anne Galvão, Priest',
     anthony_of_the_caves_monk: 'Saint Anthony of the Caves, Monk',
     anthony_zaccaria_priest: 'Saint Anthony Zaccaria, Priest',
+    antoine_chevrier_priest: 'Blessed Antoine Chevrier, Priest',
```
Anglicise his first name ([src](https://leprado.org/wp-content/uploads/2018/12/Mass-of-F.Chevrier.pdf)).

```suggestion
    anthony_chevrier_priest: 'Blessed Anthony Chevrier, Priest',
```

tukusejssirs commented on lib/locales/en.ts:
```diff
@@ -304,6 +309,7 @@ export const locale: Locale = {
     christopher_of_palestine_martyr: 'Saint Christopher, Martyr',
     ciaran_of_clonmacnoise_abbot: 'Saint Ciaran, Abbot',
     clare_of_assisi_virgin: 'Saint Clare, Virgin',
+    claudine_thevenet_religious: 'Saint Claudine Thévenet, Religious',
```
Add her religious name to the key name.

```suggestion
    marie_of_saint_ignatius_claudine_thevenet_religious: 'Saint Claudine Thévenet, Religious',
```

tukusejssirs commented on lib/locales/en.ts:
```diff
@@ -350,6 +356,7 @@ export const locale: Locale = {
       'The Dedication of the Cathedral Basilica of Saint-Denis, France',
     dedication_of_the_cathedral_of_notre_dame_de_strasbourg_france:
       'The Dedication of the Cathedral of Notre-Dame of Strasbourg',
+    dedication_of_the_cathedral_saint_john_of_lyon_france: 'The Dedication of the Primatiale Cathedral',
```
1. Rename the key to `dedication_of_the_cathedral_of_saint_john_of_lyon_france` (add missing `of_` before `saint`.
2. I believe that `Primatiale Cathedral` is a mistake. While the Cathedral is a Primatiale Cathedral, I think we should change the the English key value to `The Dedication of the Cathedral of Saint John of Lyon`.
3. According to [Wikipedia](https://en.wikipedia.org/wiki/Lyon_Cathedral), the Cathedral is dedicated to Saint John the Baptist, not Saint John of Lyon (if there is any such saint), therefore we would rename this key (and `en.ts` value) accordingly (see the suggestion below).

```suggestion
    dedication_of_the_cathedral_of_saint_john_the_baptist_lyon_france: 'The Dedication of the Cathedral of Saint John the Baptist, Lyon',
```

tukusejssirs commented on lib/locales/en.ts:
```diff
@@ -386,8 +393,10 @@ export const locale: Locale = {
     eoghan_of_ardstraw_bishop: 'Saint Eoghan, Bishop',
     ephrem_the_syrian_deacon: 'Saint Ephrem, Deacon and Doctor of the Church',
     epiphany_of_the_lord: 'The Epiphany of the Lord',
+    epipodius_and_alexander_of_lyon_martyrs: 'Saints Epipodius and Alexander, Martyrs',
```
Add ‘surname’ of Saint Epipodius to the key name.

```suggestion
    epipodius_of_lyon_and_alexander_of_lyon_martyrs: 'Saints Epipodius and Alexander, Martyrs',
```

tukusejssirs commented on lib/locales/en.ts:
```diff
@@ -420,6 +429,8 @@ export const locale: Locale = {
     frances_xavier_cabrini_virgin: 'Saint Frances Xavier Cabrini, Virgin',
     francis_borgia_priest: 'Saint Francis Borgia, Priest',
     francis_de_sales_bishop: 'Saint Francis de Sales, Bishop and Doctor of the Church',
+    francis_de_sales_bishop_patron_of_lyon_clergy:
```
`Lyon`: City or Diocese?

emagnier replied:
Diocese 👍

tukusejssirs commented on lib/locales/en.ts:
```diff
@@ -420,6 +429,8 @@ export const locale: Locale = {
     francis_borgia_priest: 'Saint Francis Borgia, Priest',
     francis_de_sales_bishop: 'Saint Francis de Sales, Bishop and Doctor of the Church',
+    francis_de_sales_bishop_patron_of_lyon_clergy:
+      'Saint Francis de Sales, Bishop and Doctor of the Church, Patron of the Lyon clergy',
```
`Lyon`: City or Diocese?

emagnier replied:
Diocese 👍

tukusejssirs replied:
Then it needs to be updated. :wink:

```suggestion
      'Saint Francis de Sales, Bishop and Doctor of the Church, Patron of the Clergy of the Archdiocese of Lyon',
```

emagnier replied:
Yes, I had only completed the key on your suggestion, but here, I simply translated literally the title from French. 

But yes, you are right, it's better this way. However, I'm tempted to leave exactly what is written in the missal in French.

tukusejssirs replied:
I think we should abide the following rules:

- for celebrations which have names officially translated (usually they are celebrated in that language), we should use the official celebration name;
- for celebrations which have no officially translated names:
  - we should do our best to translate the celebration name;
  - diocese/city names should be written always in full;
  - ideally, we should also add the country name after the diocese/city name (both in the key name and its value) which is not the case ATM.

tukusejssirs replied:
> However, I'm tempted to leave exactly what is written in the missal in French.

I absolutely agree with that! :wink:

tukusejssirs commented on lib/locales/en.ts:
```diff
@@ -507,6 +519,8 @@ export const locale: Locale = {
     innocent_xi_pope: 'Blessed Innocent XI, Pope',
     irenaeus_of_lyon_bishop: 'Saint Irenaeus, Bishop, Martyr and Doctor of the Church',
+    irenaeus_of_lyon_bishop_patron_of_diocese_of_lyon_and_companions_martyrs:
+      'Saints Ireneus, Bishop and Doctor of the Church, Principal Patron of the Archdiocese, and Companions, Martyrs',
```
1. There is always a definite article (`the`) before a name of a diocese.
2. Use the full name of the diocese (both in key and value; in `fr.ts` you should use the celebration name as it is used in the official liturgical books).
3. In English, `Irenaeus` spelling is prefered over `Ireneus`.

```suggestion
    irenaeus_of_lyon_bishop_patron_of_the_archdiocese_of_lyon_and_companions_martyrs:
      'Saints Irenaeus, Bishop and Doctor of the Church, Principal Patron of the Archdiocese of Lyon, and Companions, Martyrs',
```

emagnier replied:
I've added the diocese name both in `en` and `fr`.

tukusejssirs commented on lib/locales/en.ts:
```diff
@@ -517,12 +531,15 @@ export const locale: Locale = {
     ivan_merz: 'Blessed Ivan Merz',
     ivo_of_kermartin_priest: 'Saint Ivo, Priest',
     jacinta_marto_and_francisco_marto: 'Saints Jacinta and Francisco Marto',
+    jacques_jules_bonnaud_martyr_and_companions_martyrs: 'Blessed Jacques-Jules Bonnaud, and Companions, Martyrs',
```
Omit duplicate `martyr` title from the key.

```suggestion
    jacques_jules_bonnaud_and_companions_martyrs: 'Blessed Jacques-Jules Bonnaud, and Companions, Martyrs',
```

emagnier replied:
Key renamed to `jacques_jules_bonnaud_priest_and_companions_martyrs` + updated `En`/`Fr` locales to add the Priest title.

tukusejssirs commented on lib/locales/en.ts:
```diff
@@ -517,12 +531,15 @@ export const locale: Locale = {
     jane_frances_de_chantal_religious: 'Saint Jane Frances de Chantal, Religious',
     januarius_i_of_benevento_bishop: 'Saint Januarius, Bishop and Martyr',
     jarlath_of_tuam_bishop: 'Saint Jarlath, Bishop',
+    jean_louis_bonnard_priest: 'Saint Jean-Louis Bonnard, Priest et Martyr',
```
```suggestion
    jean_louis_bonnard_priest: 'Saint Jean-Louis Bonnard, Priest and Martyr',
```

tukusejssirs commented on lib/locales/en.ts:
```diff
@@ -857,6 +878,8 @@ export const locale: Locale = {
     pothinus_of_lyon_bishop_blandina_of_lyon_virgin_and_companions_martyrs:
       'Saints Pothinus, Bishop, Blandina, Virgin, and Companions, Martyrs',
+    pothinus_of_lyon_bishop_patron_the_city_of_lyon_blandina_of_lyon_virgin_and_companions_martyrs:
+      'Saints Pothinus, Bishop, Patron of the city of Lyon, Blandina, Virgin, and Companions, Martyrs',
```
In English, word `city`/`town` is always written with captital initial letter if it is used before a specific city/town.

```suggestion
    pothinus_of_lyon_bishop_patron_the_city_of_lyon_blandina_of_lyon_virgin_and_companions_martyrs:
      'Saints Pothinus, Bishop, Patron of the City of Lyon, Blandina, Virgin, and Companions, Martyrs',
```

tukusejssirs commented on lib/locales/en.ts:
```diff
@@ -925,10 +948,12 @@ export const locale: Locale = {
     teresa_of_jesus_of_avila_virgin: 'Saint Teresa of Jesus, Virgin and Doctor of the Church',
     teresa_of_jesus_of_los_andes_virgin: 'Saint Teresa of Jesus of Los Andes, Virgin',
     teresa_of_portugal_religious: 'Blessed Teresa of Portugal, Religious',
+    the_penitent_thief: 'The Penitent Thief',
```
No initial definite article in the key names.

Also consider using `Dismas the Good Thief` or `Dismas the Penitent Thief`.

```suggestion
    penitent_thief: 'The Penitent Thief',
```

emagnier replied:
I've renamed the key (and English locale) to `dismas_the_good_thief`, `Dismas the Good Thief`.

tukusejssirs replied:
Oh, I forgot to check that option. 🤦‍♂️ 👍

tukusejssirs commented on lib/locales/en.ts:
```diff
@@ -925,10 +948,12 @@ export const locale: Locale = {
     theodore_romzha_bishop: 'Blessed Theodore Romzha, Bishop and Martyr',
     theodosius_of_the_caves_abbot: 'Saint Theodosius of the Caves, Abbot',
     theotonius_of_coimbra_priest: 'Saint Theotonius of Coimbra, Priest',
+    therese_couderc_virgin: 'Saint Thérèse Couderc, Virgin',
```
Add first name to the key name.

Is she really celebrated as _virgin_? According to [Wikipedia](https://en.wikipedia.org/wiki/Th%C3%A9r%C3%A8se_Couderc), she was a co-founder of Sisters of the Cenacle, therefore she could be considered a religious.

```suggestion
    therese_marie_victoire_couderc_virgin: 'Saint Thérèse Couderc, Virgin',
```

tukusejssirs reviewed (changes requested):
Sorry, @emagnier, that I haven’t reviewed the PR yet.

Note that I commented only on `lib/locales/en.ts` and not on `lib/locales/fr.ts`. Some changes might be changed in `lib/locales/fr.ts` too.

> ## **Added 2 new Patron type:**
>
>     * `PatronOfTheLyonClergy`
>
>     * `PatronOfTheCityOfLyon`
>
>
> Note: adding `Patrons` (as well as `Titles`) here isn't ideal. We should think about a better way to store these kinds of metadata in the future.

Indeed. I have already thought about this. We definitely don’t want to include any specific place in the patron type name; we should use a property of the type (see #201). IMHO, all patronage titles (and regular titles overrides) should be defined in the calendars, not in the l10n files. We could even have a list of all possible patronages in `lib/catalog/martyrology.ts`. However, that would require to have a list of all patronage places used in romcal and have them localised.

emagnier committed:
data review

emagnier committed:
data review

emagnier commented:
Thank you for this detailed and quality review @tukusejssirs!! 
Absolutely no problems for the delay (and on my side, I have not yet finished addressing all the points you raised).

tukusejssirs commented on lib/locales/en.ts:
```diff
@@ -507,6 +520,8 @@ export const locale: Locale = {
     innocent_xi_pope: 'Blessed Innocent XI, Pope',
     irenaeus_of_lyon_bishop: 'Saint Irenaeus, Bishop, Martyr and Doctor of the Church',
+    irenaeus_of_lyon_bishop_patron_of_the_archdiocese_of_lyon_and_companions_martyrs:
+      'Saints Irenaeus, Bishop and Doctor of the Church, Principal Patron of the Archdiocese of Lyon, and Companions, Martyrs',
```
In `en.ts`, we don’t use `Principal Patron`, but simply `Patron`. Secondary patrons are `copatrons`.

```suggestion
      'Saints Irenaeus, Bishop and Doctor of the Church, Patron of the Archdiocese of Lyon, and Companions, Martyrs',
```

emagnier replied:
Got it 👍

tukusejssirs commented:
@emagnier, I think that this PR will be mergable after you resolve the remaining four conversations. Thanks for doing such a great job! 💯

emagnier committed:
pr feedbacks

tukusejssirs commented:
@emagnier, we’re nearly done, just [one nitpick](https://github.com/romcal/romcal/pull/421#discussion_r1374798811) is remaining. :wink:

emagnier committed:
Update lib/locales/en.ts

Co-authored-by: Tukusej’s Sirs <tukusejssirs@protonmail.com>

emagnier commented:
Thank you very much for the review @tukusejssirs! 🙏 

One last question. I think it might be worthwhile to specify the source for all these elements I added within the context of this PR: `// src: mr_fr_2014_ed2_archidiocese_of_lyon`. 

Is the abbreviation correct? (this missal is based on the second edition of the French missal (1977)

tukusejssirs commented:
> Thank you very much for the review @tukusejssirs! 🙏

You’re welcome! :wink:

> One last question. I think it might be worthwhile to specify the source for all these elements I added within the context of this PR: `// src: mr_fr_2014_ed2_archidiocese_of_lyon`.

Indeed, I have forgot about that. :+1:

> Is the abbreviation correct? (this missal is based on the second edition of the French missal (1977)

The format is undefined as of yet. We might consider the format to be `mr_${locale}_${publishYear}_ed${typicalEditionNumber}_${notes}` where `notes` is a special string to differentiate between different editions of a particular translation, like `uk` vs `us` for the English translations or the MRs published by/for a particular diocese (like in this case).

Therefore, we could use `mr_fr_2014_ed2_archidiocese_of_lyon` or simply `mr_fr_2014_ed2_lyon`. I’d probably prefer the latter, as it is shorter and it does not actually matter that `lyon` means the Archdiocese and not the City. WDYT? :thinking:

emagnier commented:
Thank you! I also prefer the short version 👍 . Indeed, Lyon here represents the diocese and not the city.

In general, there is rarely a distinct missal for a city (I mean, a distinct published or printed book, because the details for the city are generally already specified in the diocesan missal). But if that were to happen, in that case, let's stick with the short version for the diocese and use the `city` specification for the city.

emagnier committed:
specify data sources
